docs/api/spec.md

```diff
@@ -184,28 +195,67 @@ try {
 catch (err) { ... }
 ```
 
-Raise an intent and retrieve data from the IntentResolution:
+Raise an intent and retrieve either data or a channel from the IntentResolution:
 ```js
 let resolution = await agent.raiseIntent("intentName", context);
 try {
-    const result = await resolution.getResult();
-    console.log(`${resolution.source} returned ${JSON.stringify(result)}`);
+  const result = await resolution.getResult();
+  if (result && result.broadcast) { //detect whether the result is Context or a Channel
```

Comment:
Maybe not a big deal. just thinking that since `getResult()` can return a Context or a Channel now, and users will need to check which it is on their end, does it matter which property they check for on the result? This code sample checks for `broadcast`, but there is another that checks for `addContextListener`. Maybe any property unique to the type is ok, just thinking about consistency.